### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

File: scripts/upgrade [hunk 1]
```diff
@@ -76,15 +23,13 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
 	ynh_script_progression --message="Upgrading source files..." --weight=4
 
-	ynh_secure_remove --file=$final_path
-
+	ynh_secure_remove --file=$install_dir
```
Comment (alexAubin) [inline scripts/upgrade:26]: There's an inconsistency with install script for which the folder already exists ... so let's mkdir it:

```
mkdir "$install_dir"
```

Comment (alexAubin) [inline scripts/upgrade:26]:
```suggestion
	ynh_secure_remove --file=$install_dir
	mkdir "$install_dir"
```

**Comment by alexAubin** [comment]
!testme

**Pull request merged**